internal/machine/lease.go

```diff
@@ -34,7 +35,9 @@ func AcquireLeases(ctx context.Context, machines []*api.Machine) ([]*api.Machine
 	releaseFunc := func(ctx context.Context, machines []*api.Machine) {
 		for _, m := range machines {
 			if err := flapsClient.ReleaseLease(ctx, m.ID, m.LeaseNonce); err != nil {
-				fmt.Fprintf(io.Out, "failed to release lease for machine %s: %s", m.ID, err.Error())
+				if strings.Contains(err.Error(), "lease not found") {
+					fmt.Fprintf(io.Out, "failed to release lease for machine %s: %s", m.ID, err.Error())
```

Comment:
Somewhat tangential to his overall change but I think this confuses customers because we've not really educated them on what a lease is. Do we actually create if we failed to release the lease? Does it give the customer a signal that something is wrong or they need to do take action?

Reply:
Let me remove this and push it into a separate PR. 

This will just hide the lease error in the event of an "lease is not found" error, this is because the TTL expires before the operation finishes.  This is a separate issue that we are working through.  

> Do we actually create if we failed to release the lease? Does it give the customer a signal that something is wrong or they need to do take action?

The release happens at the very end, but eventually the action we will prompt the user with is to manually release the leases using the new cli command @rugwirobaker  introduced.

Reply:
👍